src/components/index.ts

```diff
@@ -49,6 +49,21 @@ export {
 } from './DisplayText';
```

Comment:
@weegrom Please export the components in this PR (Badge, Banner, Tag, Tooltip, and Visually Hidden) in this file (`src/components/index.ts`)